module1/lessons/hashes.md

```diff
@@ -35,16 +49,18 @@ Take note of:
   - What was challenging? 
   - What patterns did you observe?
   - What rules did you discover about how hashes work?
-  - What syntax did you learn for hashes?
+  - What syntax did you learn for hashes ?
   - What questions came up while you were working and how did you get those answered?
   - What questions do you still have?
   - Anything else that seems important.
 
 Resources to use while working through the exercises:
   - Your breakout room mates!
-  - Google (ex: `Ruby how to make a new hash with a default value`)
+  - Google (ex: Ruby how to make a new hash with a default value, JavaScript how to make a new object with a default value)
   - [Ruby Docs](https://ruby-doc.org/core-2.7.0/Hash.html)
```

Comment:
Need to update this link to the current version of Ruby we're using. This one is for version 2.7.0. Can just change the 2.7.0 to 3.0.0 in this link and that will work.

Reply:
Thank you 🙏🏼 - fixed it here [3be59e3](https://github.com/turingschool/curriculum-site/pull/59/commits/3be59e36e8a0380988a881c11c13f66fd4be31c1)